### PR TITLE
Add Marketplace 

### LINKED ISSUE #13: Add marketplace
Add easy to use methods for interacting with all Marketplace API(https://docs.ordinalsbot.com/marketplace)

- [x] Create structure to allow import of marketplace set of methods as a separate class
- [x] Add all methods to the sdk with types
- [x] Add test suite for `marketplace` methods
- [x] Add `testnet` support https://docs.ordinalsbot.com/api/testnet
- [x] Increase `ordinalsbot` test coverage
- [x] Update `OrdinalsBot` import to `Inscription`
- [x] Test in a new project
- [x]  Add examples to Readme
- [x] Prepare release notes and CHANGELOG

### PR DESCRIPTION
**Pull request by T1-O**
Add Market Place to SDK

Closes #13

### CODE REVIEW

File: test/integration.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ describe("order", function () {
 
   describe("create order", function () {
     it("should return a order object", async () => {
+      await delay(1000); // to avoid rate limit
```
Comment (dannybabbev) [inline test/integration.js:35]: this is not a good practice - making the tests run slow = bad develop experience. Can you avoid the rate limiter in your tests instead?

**Review by ordinariusprof — APPROVED** [review]
LGTM 🚀

File: test/integration.js (hunk 1 above)
Comment (bruffstar) [reply test/integration.js:35]: +1

Comment (T1-O) [reply test/integration.js]: @dannybabbev @bruffstar Removed the delay and refactored inscription test cases.

**Pull request merged**